Tests/SkillFunctionalTests/SimpleHostBotToDialogSkillTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using TranscriptTestRunner;
+using TranscriptTestRunner.XUnit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SkillFunctionalTests
+{
+    public class SimpleHostBotToDialogSkillTest

```

Comment:
Refactor this class to derive from ScriptTestBase from scenariosStaging and remove redundant code before merging it

Reply:
Done. Thanks!